tests/test_ascii_art.py

```
@@ -0,0 +1,61 @@
+import unittest
+from src.utils.ascii_art import convert_pixel_art_to_ascii, load_ascii_art, display_ascii_art
+from src.services.combat import calculate_damage, process_status_effects
+from src.models.character import Player, Enemy
+from src.models.character_classes import CharacterClass
+from src.models.skills import Skill
+from src.models.status_effects import BLEEDING, POISONED, WEAKENED, BURNING, CURSED
```

Ruff (0.8.0)

`src.models.status_effects.WEAKENED` imported but unused

Remove unused import

(F401)